internal/manager/config.go

```diff
@@ -109,16 +110,6 @@ type Config struct {
 // Controller Manager - Config - Methods
 // -----------------------------------------------------------------------------
 
-// Validate validates the config. With time this logic may grow to invalidate
-// incorrect configurations.
-func (c *Config) Validate() error {
```

Comment:
This would still be useful if someone was to test flags against each other, e.g. for being mutually exclusive (like I intended to do in one of the subsequent PRs :) )

Reply:
So what's the conclusion on this one :)? Can we keep it or you have an alternative to propose?

Reply:
I think we may add it back when it's needed (there's no point in leaving it in this PR just to be empty). Maybe let's just leave a comment that it should be used for cross-variable checks?

Reply:
I'd suggest to leave it "empty" with a comment that this can be used for more complex flags validations that the newly introduced `NewValidatedValue...` wouldn't solve e.g. flags cross validation. WDYT?

Reply:
[e08ba10](https://github.com/Kong/kubernetes-ingress-controller/pull/3415/commits/e08ba103cf45b9a71f5ba502adf24f4ca5c36f9f)